test/WebSites/FiltersWebSite/Controllers/SpecificFormattersController.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.AspNet.Mvc.Formatters;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace FiltersWebSite
```

Comment:
Can you put this in `BasicWebSite` instead? We might mothball this site. This is a reasonable test, and I don't want it to get lost in the scrum.